spec/services/authentication/authenticator_spec.rb

```diff
@@ -150,6 +150,24 @@
         end.to change { user.reload.apple_username }.to("newname")
       end
 
+      # rubocop:disable RSpec/AnyInstance
```

Comment:
I did my best to avoid this but the suggested approach using doubles didn't work for me. We do use this in a few different places so it may not be that bad to add the exception here too. Otherwise I'd really appreciate a suggestion to work around this 🙂